### PR TITLE
Feature player id api

### PR DESCRIPTION
**Pull request by Nilint1**
Adds the ability to get players and goalies by id, for a specific league.
Added documentation.

### CODE REVIEW

File: pages/api/v1/goalies/ratings/index.ts [hunk 1]
```diff
@@ -2,8 +2,8 @@ import Cors from 'cors';
 import { NextApiRequest, NextApiResponse } from 'next';
 import SQL from 'sql-template-strings';
 
-import { query } from '../../../../lib/db';
-import use from '../../../../lib/middleware';
+import { query } from '../../../../../lib/db';
```
Comment (Nilint1) [inline pages/api/v1/goalies/ratings/index.ts:5]: moved original stats and ratings files into a folder and changed names to "index.ts".  This allows for the use of [id] in all cases.

File: pages/api/v1/goalies/ratings/[id].ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  AND corrected_player_ratings.SeasonID = player_master.SeasonID
+  AND corrected_player_ratings.LeagueID = player_master.LeagueID
+  INNER JOIN team_data
+  ON player_master.TeamID = team_data.TeamID
+  AND corrected_player_ratings.SeasonID = team_data.SeasonID
+  AND corrected_player_ratings.LeagueID = team_data.LeagueID
+  WHERE corrected_player_ratings.LeagueID=${+league}
+  AND corrected_player_ratings.G=20
+  AND player_master.TeamID>=0
+  AND corrected_player_ratings.PlayerID = ${+id}
```
Comment (Nilint1) [inline pages/api/v1/goalies/ratings/[id].ts:34]: this is the only change.  Removes the season from the where statement (unless it is requested) and add player id to the where statement.

File: pages/api/v1/goalies/ratings/[id].ts [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+      team: player.Abbr,
+      position: 'G',
+      blocker: player.Blocker,
+      glove: player.Glove,
+      passing: player.GPassing,
+      pokeCheck: player.GPokecheck,
+      positioning: player.GPositioning,
+      rebound: player.Rebound,
+      recovery: player.Recovery,
+      puckhandling: player.GPuckhandling,
```
Comment (esilverm) [inline pages/api/v1/goalies/ratings/[id].ts:58]: Names should be in camelCase

**Review by esilverm — APPROVED** [review]
Just change up the camelcase and it is good to go

File: pages/api/v1/goalies/ratings/[id].ts (hunk 3 above)
Comment (tallsmallone) [reply pages/api/v1/goalies/ratings/[id].ts:58]: This is a direct copy from the original files.  Do you want me to update them all?  Also, which line specifically?  Puckhandling?

Comment (esilverm) [reply pages/api/v1/goalies/ratings/[id].ts:58]: ah if so ill allow it 👍

**Pull request merged**